ScoutSuite/frontend/src/components/GetResourceLink/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Link } from 'react-router-dom';
+import GetResourceName from '../GetResourceName/index';
+
+const propTypes = {
+  service: PropTypes.string.name,
+  resource: PropTypes.string.isRequired,
+  id: PropTypes.string.isRequired,
+};
+
+const GetResourceLink = ({ service, resource, id }) => {
+  return (
+    <Link to={`/services/${service}/resources/${resource}/${id}`}>
+      <GetResourceName
```

Comment:
another thing I thought of, since they're components, I feel like simply `ResourceLink` and `ResourceName` would be better names.

Reply:
Fixed and added a name props for resources where we already know the name